pydis_site/hosts.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+from django_hosts import host, patterns
+
+host_patterns = patterns(
+    '',
+    # > | Subdomain | URL Module   | Host entry name |
+    host(r'admin', 'pydis_site.apps.admin.urls', name="admin"),
+    host(r'api', 'pydis_site.apps.api.urls', name='api'),
+    # host(r"staff",    "pydis_site.apps.staff",  name="staff"),
+    # host(r"wiki",     "pydis_site.apps.wiki",   name="wiki"),
+    # host(r"ws",       "pydis_site.apps. ws",     name="ws"),
```

Comment:
I kinda feel like we can add these when we need them, not a fan of this.

Reply:
Was already there. I can remove it if you like!